istioctl/cmd/proxystatus_test.go

```diff
@@ -54,6 +54,14 @@ func TestProxyStatus(t *testing.T) {
 			args:          strings.Split("proxy-status serviceaccount/sleep", " "),
 			wantException: true,
 		},
+		{ // case 8: new -o argument with json
```

Comment:
Can you please also add a test to cover the correct json format case?

Reply:
@hanxiaop  case 8 covers the "correct" format in that if the command arguments are valid we would expect and empty string result. I based this based on the cases 0 & 1 which just compared correct output as only the table header row but no additional proxy data. In the json case we would expect this to be an empty string as there is no header row to print prior to the json data.